***Final Project Notebook***

Name: Pooya Noori

**Introduction:**

For this project i chose to find best neighborhood for living (buy house) in pittsburgh in terms of low population, high homevalue, and median income.the reason is beacause we are planing to buy a house with my family, and that is why choosing neighborhood based on living is my option.


**The Metric**

My goal is to find neighbordhood with the great and most balanced average cost of living, safety, and housing. i have some resources like https://www.niche.com/places-to-live/search/best-neighborhoods/m/pittsburgh-metro-area/https://www.niche.com/places-to-live/search/best-neighborhoods/m/pittsburgh-metro-area/ that is showing all aspects of neighborhood based on living.

Here is my result(DatasetLink):
https://docs.google.com/spreadsheets/d/1mS9AC4HDMfSwb5Or11GlwJnyRttmNBb7iHePi_KeIM8/edit#gid=0

In [23]:
import pandas as panda
#read in the data from the file
data = panda.read_csv('housing - Sheet1.csv')
data.head()

Rank         Neighborhood Population Home Value Median Income
0     1        Regent Square      4,067   $300,900       $94,953
1     2         Point Breeze      5,555   $276,757      $109,215
2     3            Shadyside     13,593   $290,693       $53,305
3     4        Highland Park      6,854   $150,738       $62,251
4     5  Squirrel Hill South     15,572   $237,164       $65,941

as you can see based on my resserch this is what ia got.
To me regent square is number 1 when it comes to living. based on most balacned neigberhood that you can imagine of what i mean by that is look at the population. it hhas a lowest population, median income is average and homevalues are resonable and if you search it on the map it is 20 minitest away from downtown as well. not saying that all other negibehood that i listed here which are (Point Breeze	5,555	$276,757	$109,215
2	3	Shadyside	13,593	$290,693	$53,305
3	4	Highland Park	6,854	$150,738	$62,251
4	5	Squirrel Hill South
thease are also great options to live in pittsburgh, but as far as number one is regent square.